File: Languages/Java/Keypad.java
package Languages.Java;
import java.util.Scanner; 

public class Keypad {
    private Scanner input; // object created from the scanner class 

    // constructor 
    public Keypad() {
        input = new Scanner(System.in); 
    }

    // method to get the input which is an integer 
    public int getInput() {
        return input.nextInt(); 
    }
    
}

File: Languages/Golang/Keypad.go
package main

import (
	"bufio"
	"os"
	"strconv"
)

type Keypad struct{}

func (k *Keypad) GetInput() int {
	reader := bufio.NewReader(os.Stdin)
	input, _ := reader.ReadString('\n')
	input = input[:len(input)-1]
	value, _ := strconv.Atoi(input)
	return value

}

func main() {}
